mautrix_telegram/web/provisioning/spec.yaml

```diff
@@ -24,6 +24,20 @@ tags:
 - name: Bridging
 
 paths:
+  /bridge:
+    get:
+      operationId: get_bridge
+      summary: Get the bridge's information
+      tags: [Misc]
```

Comment:
The "Misc" tag should be added to the tags array above paths

Reply:
Added